cli/src/errors/details.rs

```diff
@@ -48,6 +48,9 @@ Please ensure you have permissions to edit your environment variables."
 
     #[error("Could not install CLI. {}", .msg)]
     CLIInstallError { msg: String },
+
+    #[error("Could not read cli configuration CLI. {}", .msg)]
+    CLIConfigError { msg: String },
```

Comment:
```suggestion
    CliConfigError { msg: String },
```

And while you're here if you can rename the one above too (https://meteor.slack.com/archives/C013EMYJ48J/p1590692356033300)